src/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValue.ts

```diff
@@ -1,18 +1,24 @@
 import * as Globalize from 'globalize';
-import { DynamicFacetValueRenderer } from './DynamicFacetValueRenderer';
-import { DynamicFacet } from '../DynamicFacet';
 import { FacetValueState } from '../../../rest/Facet/FacetValueState';
 import { IAnalyticsDynamicFacetMeta, analyticsActionCauseList } from '../../Analytics/AnalyticsActionListMeta';
 import { l } from '../../../strings/Strings';
 import { IRangeValue, RangeType } from '../../../rest/RangeValue';
 import { FacetType } from '../../../rest/Facet/FacetRequest';
+import { IDynamicFacet } from '../DynamicFacetInterface';
+import { IFacetResponseValue } from '../../../rest/Facet/FacetResponse';
+
+export interface ValueCreator {
```

Comment:
It's a good convention to start interface names with a capital `I` to make them easier to distinguish from class instances when reading the code.